indeex.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="stylesheet.css" />
+    <header>
+      <div></div>
+      <div></div>
+      <div class="login">
+        <h3><a href="#">login</a></h3>
+      </div>
+    </header>
+  </head>
+  <body>
+    <!-- container practice -->
+
+    <div class="main-container">
+      <div class="personalinfo">
```

Comment:
Please update the class names to kebab case.

Reply:
resolved!